src/routes/safe/components/Balances/SendModal/screens/SendFunds/TokenSelectField/index.jsx

```diff
@@ -16,26 +16,27 @@ import { formatAmount } from '~/logic/tokens/utils/formatAmount'
 import { selectedTokenStyles, selectStyles } from './style'
 
 type SelectFieldProps = {
-  tokens: List<Token>,
   classes: Object,
   initialValue: string,
+  isValid: boolean,
+  tokens: List<Token>,
 }
 
 type SelectedTokenProps = {
-  tokenAddress?: string,
   classes: Object,
+  tokenAddress?: string,
   tokens: List<Token>,
 }
 
-const SelectedToken = ({ tokenAddress, tokens, classes }: SelectedTokenProps) => {
+const SelectedToken = ({ classes, tokenAddress, tokens }: SelectedTokenProps) => {
```

Comment:
are we sorting props by alphabet now? :D

Reply:
that's someone IDE setup, probably an extension. Not sure. @gabitoesmiapodo ?

Reply:
Not sure about this... I sometimes will sort manually some props, but only if there is one prop by line distributed among several lines. I don't think this is one of those situations, though.

I'll quickly add a few sort related rules in a PR so it's the same for everybody.

Reply:
Added some linting rules for this at https://github.com/gnosis/safe-react/pull/614